packages/@lwc/integration-karma/scripts/karma-configs/test/sauce.js

```diff
@@ -51,26 +51,32 @@ const SAUCE_BROWSERS = [
 
     // Compat browsers
     {
-        label: 'sl_ie11',
-        browserName: 'internet explorer',
-        version: '11',
+        label: 'sl_edge_compat',
+        browserName: 'MicrosoftEdge',
+        version: '18',
         compat: true,
         nativeShadowCompatible: false,
     },
     {
         label: 'sl_chrome_compat',
         browserName: 'chrome',
-        version: '59',
+        version: 'latest-2',
         compat: true,
-        nativeShadowCompatible: false,
+        nativeShadowCompatible: true,
+    },
+    {
+        label: 'sl_firefox_compat',
+        browserName: 'FirefoxESR',
+        version: 'latest',
+        compat: true,
+        nativeShadowCompatible: true,
     },
     {
         label: 'sl_safari_compat',
         browserName: 'safari',
-        version: '10',
-        platform: 'OS X 10.11',
+        version: '13',
         compat: true,
-        nativeShadowCompatible: false,
+        nativeShadowCompatible: true,
```

Comment:
`COMPAT` for Karma is now these browsers... up for debate exactly which ones we test here.